app.py

```diff
@@ -8,7 +8,7 @@ def chat(user_input):
     try:
         response = message_to_bot(user_input, clf, learn_response)
     except:
-        return jsonify({'message': "Unable to get response"}, 500)
+        return jsonify({'message': ('Unable to get response', 0)}, 500)
     
     return jsonify({'message': response}, 200)
```

Comment:
The tuple response is necessary here, and the response cannot be hardcoded, rather should be the correct value of `learn_response`

Reply:
@vishakha-lall Is the value of `learn_response` sufficient here? It seems vague now but I think it should be sufficient after the symbolic names have been created, right?